meson_options.txt

```diff
@@ -26,3 +26,4 @@ option('update_server', type: 'string', value: 'https://aegisub-updates.redvice.
 option('update_url', type: 'string', value: '/trunk', description: 'Base path to use for the update checker')
 
 option('build_osx_bundle', type: 'boolean', value: false, description: 'Package Aegisub.app on OSX')
+option('bash_completion', type: 'boolean', value: true, description: 'Install bash shell completions')
```

Comment:
Won't this run by default on Windows? I don't think I can merge this as-is.

Reply:
I don't have a Windows system to test. I added a new commit to disable this for Windows.